gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -448,6 +449,33 @@ local key_params = {
 	walkkey = Key("+walk", "WALK")
 }
 
+-- handle looking at doors
+function HUDDrawTargetIDDoors(tData)
+	local client = LocalPlayer()
+	local ent = tData:GetEntity()
+
+	if not IsValid(client) or not client:IsTerror() or not client:Alive()
+	or not IsValid(ent) or not ent:IsDoor() or ent:IsDoorForceclosed() or tData:GetEntityDistance() > 100 then
```

Comment:
Two times `not IsValid`, this can be simplified with brackets

Reply:
Forceclosed or ForceClosed?

Reply:
I don't thinkt it is better or more readable if combined in brackets as `not (IsValid(client) and IsValid(ent))`. I personally like the current approach better from a readability standpoint.

Forceclosed since the internal variable is named that way